src/components/dashboard/data/utils.js

```diff
@@ -1,4 +1,31 @@
 import { ASSIGNMENT_TYPES } from '../../enterprise-user-subsidy/enterprise-offers/data/constants';
+import {
+  LEARNER_ACKNOWLEDGED_ASSIGNMENT_CANCELLATION_ALERT,
+  LEARNER_ACKNOWLEDGED_ASSIGNMENT_EXPIRATION_ALERT,
+} from '../main-content/course-enrollments/data/constants';
+
+export function getIsActiveExpiredAssignment() {
+  // if item has never been set, user has not dismissed any expired assignments
+  const lastExpiredAlertDismissedTime = global.localStorage.getItem(LEARNER_ACKNOWLEDGED_ASSIGNMENT_EXPIRATION_ALERT);
+  if (lastExpiredAlertDismissedTime === null) {
+    return true;
+  }
+  const currentDate = new Date();
+  return (currentDate > new Date(lastExpiredAlertDismissedTime));
```

Comment:
Won't this always be true?  Down in `handleOnCloseExpiredAlert`, we're setting this local storage key to have a value of the current date.  Maybe this function should do a similar thing to what's going on below in `getIsActiveCancelledAssignment()`, where we compare the latest cancelled action time to whatever time is currently in global storage (i.e. the last time the cancelled alert was dismissed).  Lemme know if I'm mis-understanding something.